### PR TITLE
Shahenda

### PR DESCRIPTION
**Pull request by shahendaeldeeb**
Updating freska migrations.

### CODE REVIEW

File: migrate.go [hunk 1]
```diff
@@ -57,27 +58,27 @@ func migrateDB(config map[string]string) {
...
 				log.Printf("Running database creation migration: %s", file)
 			}
-
 			// Execute this sql file against the database
 			result, err := runCommand("psql", args...)
 			if err != nil || strings.Contains(string(result), "ERROR") {
 				if err == nil {
 					err = fmt.Errorf("\n%s", string(result))
-				}
+				}else {
```
Comment (mcherri) [inline migrate.go]: Is this a bug in Fragmenta? Anyway the else is not well formatted.

**Review by mcherri — CHANGES_REQUESTED** [review]
Please fix minor issue

File: migrate.go [hunk 2]
```diff
@@ -68,8 +68,7 @@ func migrateDB(config map[string]string) {
 			if err != nil || strings.Contains(string(result), "ERROR") {
 				if err == nil {
 					err = fmt.Errorf("\n%s", string(result))
-				}else {
-
+				} else {
```
Comment (mcherri) [inline migrate.go]: Why are we only breaking from this else only?

**Review by mcherri — CHANGES_REQUESTED** [review]
A small question

File: migrate.go (hunk 2 above)
Comment (shahendaeldeeb) [reply migrate.go]: As this is the only else inside loop of sql files

Comment (mcherri) [reply migrate.go]: We are testing for `err == nil`, which will never happen. If this is not true then we need to talk.

File: migrate.go [hunk 3]
```diff
@@ -68,12 +68,12 @@ func migrateDB(config map[string]string) {
...
 				if err == nil {
 					err = fmt.Errorf("\n%s", string(result))
-				} else {
-					// If at any point we fail, log it and break
-					log.Printf("ERROR loading sql migration:%s\n", err)
-					log.Printf("All further migrations cancelled\n\n")
-					break
 				}
+				// If at any point we fail, log it and break
+				log.Printf("ERROR loading sql migration:%s\n", err)
```
Comment (mcherri) [inline migrate.go]: I think this line only belongs to the else because `err` can be `nil` sometimes.

**Review by mcherri — CHANGES_REQUESTED** [review]
A minor change.

**Review by mcherri — APPROVED** [review]
:100:

**Pull request merged**